app/import.go

```diff
@@ -1702,17 +1701,13 @@ func (a *App) OldImportChannel(channel *model.Channel) *model.Channel {
 	}
 }
 
-func (a *App) OldImportFile(timestamp time.Time, file io.Reader, teamId string, channelId string, userId string, fileName string) (*model.FileInfo, error) {
-	buf := bytes.NewBuffer(nil)
-	io.Copy(buf, file)
-	data := buf.Bytes()
-
-	fileInfo, err := a.DoUploadFile(timestamp, teamId, channelId, userId, fileName, data)
+func (a *App) OldImportFile(timestamp time.Time, file io.ReadCloser, teamId string, channelId string, userId string, fileName string) (*model.FileInfo, error) {
+	fileInfo, err := a.DoUploadFile(timestamp, teamId, channelId, userId, fileName, file)
 	if err != nil {
 		return nil, err
 	}
 
-	img, width, height := prepareImage(data)
+	img, width, height := prepareImage(file)
```

Comment:
Given that we pass `file`, an `io.ReadCloser` to `DoUploadFile`, at this point the reader is at `EOF`, and `prepareImage` has nothing to work with.